WebView/ViewController.swift

```diff
@@ -6,14 +6,39 @@
 //
 
 import UIKit
+import WebKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, WKNavigationDelegate {
+    var webView: WKWebView!
+    
+    override func loadView() {
+        webView = WKWebView()
+        webView.navigationDelegate = self
+        view = webView
+    }
+        
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Open", style: .plain, target: self, action: #selector(openTapped))
+        guard let url = URL(string: "https://www.redspace.com") else { return }
+        webView.load(URLRequest(url: url))
+        webView.allowsBackForwardNavigationGestures = true
+    }
+    @objc func openTapped() {
+        let ac = UIAlertController(title: "Open page...", message: nil, preferredStyle: .actionSheet)
+        ac.addAction(UIAlertAction(title: "apple.com", style: .default, handler: openPage))
+        ac.addAction(UIAlertAction(title: "google.com", style: .default, handler: openPage))
+        ac.addAction(UIAlertAction(title: "Cancel", style: .cancel))
+        ac.popoverPresentationController?.barButtonItem = navigationItem.rightBarButtonItem
+        present(ac, animated: true)
+    }
+    func openPage(action: UIAlertAction) {
+        guard let actionTitle = action.title else { return }
```

Comment:
Minor: if it makes sense you can combine the guard statements, especially if you want to do the same thing if it fails (in this case `return`) 
```swift
guard let actionTitle = action.title,
          let url = URL(string: "https://" + actionTitle) else { return }
```